SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialActorUtils.h

```diff
@@ -44,6 +44,32 @@ inline FString GetConnectionOwningWorkerId(const AActor* Actor)
 	return FString();
 }
 
+// Effectively, if this Actor is in a player hierarchy, get the PlayerController entity ID.
+inline Worker_PartitionId GetConnectionOwningPartitionId(const AActor* Actor)
```

Comment:
This function only seem valid to call on a server, append Server to function name?